tests/dCommonTests/dEnumsTests/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+set(DENUMS_TESTS
+	"MagicEnumTests.cpp"
+)
+
+# Get the folder name and prepend it to the files above
+get_filename_component(thisFolderName ${CMAKE_CURRENT_SOURCE_DIR} NAME)
+list(TRANSFORM DENUMS_TESTS PREPEND "${thisFolderName}/")
+
+# Export to parent scope
+set(DENUMS_TESTS ${DENUMS_TESTS} PARENT_SCOPE)
```

Comment:
We don't write these like this right now, so for consistency this should follow how the other folder names are gotten (CMakeLists.txt in this directory just globs its files and sends to PARENT_SCOPE, then the parent scope prepends the folder name it just added)

Reply:
Currently, the tests are different and other tests cmakeslists are like this.
Leave it as is and we can make them consistent in a separate PR